airbyte-scheduler/scheduler-persistence/src/main/java/io/airbyte/scheduler/persistence/job_error_reporter/SentryErrorReportingClient.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.scheduler.persistence.job_error_reporter;
+
+import io.airbyte.config.FailureReason;
+import io.airbyte.config.StandardWorkspace;
+import io.sentry.Sentry;
+import io.sentry.SentryEvent;
+import io.sentry.protocol.Message;
+import io.sentry.protocol.SentryException;
+import io.sentry.protocol.SentryId;
+import io.sentry.protocol.User;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class SentryErrorReportingClient implements ErrorReportingClient {
+
+  private static final Logger LOGGER = LoggerFactory.getLogger(LoggingErrorReportingClient.class);
+
+  public SentryErrorReportingClient(final String sentryDSN) {
+    Sentry.init(options -> {
+      options.setDsn(sentryDSN);
+      options.setEnableUncaughtExceptionHandler(false);
+    });
+  }
+
+  @Override
+  public void report(final StandardWorkspace workspace,
+                     final FailureReason failureReason,
+                     final String dockerImage,
+                     final Map<String, String> metadata) {
+    final SentryEvent event = new SentryEvent();
+
+    // airbyte/source-xyz:1.2.0 -> source-xyz@1.2.0
+    final String release = dockerImage.replace(":", "@").substring(dockerImage.lastIndexOf("/") + 1);
```

Comment:
is removing the `/` cosmetic? why not just keep it to stay true to the actual image name and to avoid bugs?